src/spinner/__tests__/component.test.js

```diff
@@ -29,9 +29,7 @@ describe('Spinner', () => {
 
   test('component overrides', () => {
     const overrides = {
-      Svg: jest
-        .fn()
-        .mockImplementation(({children}) => <span>{children}</span>),
+      Svg: jest.fn().mockImplementation(({children}) => <svg>{children}</svg>),
```

Comment:
the `span` usage caused a warning in tests

Reply:
probably those children are allowed only inside `svg` tag